kafka-connect-neo4j/src/test/kotlin/streams/kafka/connect/source/Neo4jSourceTaskTest.kt

```diff
@@ -274,6 +283,16 @@ class Neo4jSourceTaskTest {
                 |       key1: "value1",
                 |       key2: "value2"
                 |   } AS map,
+                |   {
+                |       key1: {
+                |           subKey1: "value",
+                |           subKey2: "value"
+                |       },
+                |       key2: {
+                |           subKey1: "value",
+                |           subKey2: true
+                |       }
+                |   } AS nested,
```

Comment:
I think there is a misunderstanding a `Map` can have only values of the same simple type, like for instance`Map<String, Integer>` or `Map<String, String>`.
The structure that you are expressing here is not a `Map`, but a `Struct`.